config/suppressions.xml

```
@@ -15,4 +15,7 @@
             files="GeneratePatchFileLauncher|GeneratePatchFileWithGitCommandLauncher\.java"/>
     <suppress checks="UncommentedMain"
             files="GeneratePatchFileLauncher|GeneratePatchFileWithGitCommandLauncher\.java"/>
+    <suppress checks="AnonInnerLength"
```

don't suppress errors

---

please remove this

---

removed